sdk/cosmosdb/cosmos/test/public/integration/query.spec.ts

```diff
@@ -232,22 +239,150 @@ describe("Test Index metrics", function (this: Suite) {
       }
     }
   }
-  async function setupContainer(datbaseName: string, collectionId: string, throughput?: number) {
-    const database = await getTestDatabase(datbaseName);
+});
 
-    const collectionDefinition = {
-      id: collectionId,
-      partitionKey: {
-        paths: ["/pk"],
-      },
-    };
-    const collectionOptions = { offerThroughput: throughput };
+describe("Test RU Capping query", function (this: Suite) {
+  beforeEach(async function () {
+    await removeAllDatabases();
+  });
 
-    const { resource: createdCollectionDef } = await database.containers.create(
-      collectionDefinition,
-      collectionOptions
+  it("For Single partition query", async function () {
+    const collectionId = "testCollection4";
+    const containerDefinition = { id: collectionId };
+    const containerOptions1 = { offerThroughput: 4000 };
+    const createdContainerSinglePartition = await await getTestContainer(
+      "RU Capping test db for single partition",
+      undefined,
+      containerDefinition,
+      containerOptions1
     );
-    const createdContainer = database.container(createdCollectionDef.id);
-    return createdContainer;
-  }
+
+    createdContainerSinglePartition.items.create({ id: "myId1", pk: "pk1", name: "test1" });
+    createdContainerSinglePartition.items.create({ id: "myId2", pk: "pk2", name: "test2" });
+    createdContainerSinglePartition.items.create({ id: "myId3", pk: "pk2", name: "test2" });
+
+    const query1 = "SELECT * from " + collectionId + " where " + collectionId + ".name = 'test2'";
+    const queryOptions: FeedOptions = { maxItemCount: 1 };
```

Comment:
Add similar options for a bigger (ideally more than 100) `maxItemCount` as well, 1 mighty not be representive for all cases.